### DISCUSSION TITLE
small displays

### DISCUSSION THREAD

**Discussion by minnten** [KlipperScreen/KlipperScreen#30] [Show and tell]
Hello,

at first, this project is great!!, thanks for working on it.
As I had a 7" display with the same resolution(1024*600) as the main distributor, I tried this at first.
But for me a smaller display would be better and one on the GPIOs would be perfect, because of the smaller space needed and lesser cabels. So I read in the changelog, that since yesterday other resolutions are possible too and I had to check this.

Here is what I got:
![grafik](https://user-images.githubusercontent.com/50588960/101700095-be6b7080-3a7c-11eb-80e0-d47c8e5af197.png)
![grafik](https://user-images.githubusercontent.com/50588960/101700117-c9260580-3a7c-11eb-9038-b3bc8336824f.png)
![grafik](https://user-images.githubusercontent.com/50588960/101700133-cfb47d00-3a7c-11eb-978e-bd21a77a6c62.png)

For me it is the perfect solution.

Greetings 
Minnten

**Comment by michelluccote** [comment]
I'm running two printers with the 3.5" displays. Works really well but I do find the fonts an especially the icons bit small. The worst are the small buttons on the upper corners. Still completely function and love it but a few tweaks would top it of nice. One question. How do you get the image on the status page? Am I missing an option in my slicer? I just get a generic STL icon.

![IMG_2323](https://user-images.githubusercontent.com/70981353/108171604-9ae22900-70b0-11eb-8457-e47860ef44b8.jpeg)

**Reply by minnten** [reply]
@michelluccote 
Here is a Link, where you can find the solution:
https://docs.mainsail.xyz/quicktips/thumbnails

**Reply by Fortunatto** [reply]
> I'm running two printers with the 3.5" displays. Works really well but I do find the fonts an especially the icons bit small. The worst are the small buttons on the upper corners. Still completely function and love it but a few tweaks would top it of nice. One question. How do you get the image on the status page? Am I missing an option in my slicer? I just get a generic STL icon.
> 
> ![IMG_2323](https://user-images.githubusercontent.com/70981353/108171604-9ae22900-70b0-11eb-8457-e47860ef44b8.jpeg)

Hello! Can you share your tweaks to make fonts and elements more readable?

**Comment by alfrix** [comment]
The current interface is much better for small small screens with larger fonts, closing as resolved
